src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/OptimizedInboxTextEncoder.Ascii.cs

```diff
@@ -87,7 +87,7 @@ internal void PopulatePreescapedData(in AllowedBmpCodePointsBitmap allowedCodePo
             {
                 this = default; // clear all existing data
 
-                Span<char> tempBuffer = stackalloc char[8] { '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0' };
+                Span<char> tempBuffer = ['\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0'];
```

Comment:
This project does build for .NET Framework:

https://github.com/dotnet/runtime/blob/31bc167803abe8381c6ecdfec1f7935dead2df10/src/libraries/System.Text.Encodings.Web/src/System.Text.Encodings.Web.csproj#L4

So I think the stackalloc there is becoming a heap alloc.

Reply:
Yes, this line should be reverted; good catch.

Reply:
```suggestion
                Span<char> tempBuffer = stackalloc char[8] { '\0', '\0', '\0', '\0', '\0', '\0', '\0', '\0' };
```